pages.de/common/kubectl-get.md

```diff
@@ -9,23 +9,23 @@
 
 - Frage alle Nodes in einem bestimmten Namespace ab:
 
-`kubectl get nodes --namespace {{namespace}}`
+`kubectl get nodes -n {{namespace}}`
 
 - Frage alle Pods in einem bestimmten Namespace ab:
 
-`kubectl get pods --namespace {{namespace}}`
+`kubectl get pods -n {{namespace}}`
 
 - Frage alle Deployments in einem bestimmten Namespace ab:
 
-`kubectl get deployments --namespace {{namespace}}`
+`kubectl get deployments -n {{namespace}}`
```

Comment:
Why do you change it to the short option here and to the long option in the grep page?

Reply:
We recently added clear guidelines for handling this in the style guide. In commands like this, we should use the long option (if it exists) and optionally highlight the short options as mnemonics. The English page seems to use short options for all examples, I would suggest updating it to long options and then highlight `n` using mnemonics.